JumboCode/english-at-large
# Yotsuuye/eng 24 displayfilter books functionality

# Description

Completed the front-end section where the books are displayed. We did the styling as well as the functionality of the filtering section. 

## Issues






Issue #24

## Screenshots

<img src="https://github.com/user-attachments/assets/b869137f-d62d-41ff-9d06-34b11445e739" width=300>
<img src="https://github.com/user-attachments/assets/f6320bd8-d976-4dec-8860-604a9edfa573" width=300>
<img src="https://github.com/user-attachments/assets/2d2568c4-58b0-4cd4-91c6-3a1384082aa0" width=300>




## Test

- Go to the display page from the main page
- When you click on the button, the page should populate with all the books sorted by title
- Click on the filter icon and select your desired filters
- When you click reset, it should clear all of your filters and default the sort to "By Title"
- When you click the submit button, the filters should apply and only show books you want to see
- If you go back to the filter section and click on the "x", the filters should revert back to what you had previously.

## Possible Downsides or 

As of now we haven't done extensive testing on the sort by date yet. We will get on that soon.

## Additional Documentations
